addon/components/flash-message.js

```diff
@@ -107,5 +107,11 @@ export default Component.extend({
     if (flash) {
       flash.destroyMessage();
     }
+  },
+
+  actions: {
+    close() {
+      this._destroyFlashMessage();
```

Comment:
I like the idea of this!
If it's stripped when this package is packaged (production build) then this one would be stripped out, hrm :/

How do other addons add warnings for users of the addon?